drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
Looking more into `RigidBodySystem`, I notice it contains a pointer to a single `RigidBodyTree`. For scalability concerns, I don't think it would be a appropriate to model multiple robots using a single `RigidBodyTree` since that may result in some really huge matrices. This further justifies the inclusion of a `robot_name` parameter in method `AddRobotFromSDFFile()`.

If we do enforce a 1:1 relationship between the number of `RigidBodySystem` objects and the number of robots being simulated / controlled, perhaps the names of `AddRobotFromXXFile()` should be changed to `LoadRobotFromXXFile()`, and to throw an exception if these methods are called more than once.